core/io/file_access_pack.cpp

```diff
@@ -197,7 +197,7 @@ bool PackedSourcePCK::try_open_pack(const String &p_path, bool p_replace_files,
 	// Search for the header at the start offset - standalone PCK file.
 	f->seek(p_offset);
 	uint32_t magic = f->get_32();
-	if (magic == PACK_HEADER_MAGIC) {
+	if (magic == PackedData::SIGNATURE) {
```

Comment:
`PackedData` is a common class, it's not specific to the PCK (also used for ZIP), this signature probably should be in `PackedSourcePCK`.